calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -1636,6 +1639,51 @@ public String merge(String siteId, Element root, String archivePath, String from
 		return results.toString();
 	}
 
+	/*
+	 * Look at the properties of the event and determine if it should be merged or ignored
+	 */
+	public boolean shouldMergeEvent(Element el)
```

Comment:
Can this be private? Also, I'd just go for a modern formatting style in this method. Curly braces at the end of the if statement and method declaration. You're mixing and matching a bit here anyway, so you may as well make it one style